src/rgw/rgw_sal_motr.cc

```diff
@@ -898,7 +901,9 @@ int MotrBucket::merge_and_store_attrs(const DoutPrefixProvider *dpp, Attrs& new_
   for (auto& it : new_attrs)
     attrs[it.first] = it.second;
 
-  return put_info(dpp, y, ceph::real_time());
+  // "merge_and_store_attrs" is meant to updating existing metadata, 
+  // hence explicitly passing false. Passing "yield" was incorrect
+  return put_info(dpp, false, ceph::real_time());
```

Comment:
so only during create bucket, this bool is set to false, so update route is ruled out. and then any operation on bucket, like elabling versioning, tagging, set quota, it should be false(default), so bucket attribute update is allowed for such APIs!